rust/wasmtime_bindings/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "wasmtime_bindings"
+version = "0.1.0"
+edition = "2021"
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+cxx = { version = "1.0.73", features = ["c++20"] }
+wasmtime = { version = "0.40.0", features = ["async"] }
+wasmtime-wasi = { version = "0.40.0", features = ["tokio"] }
```

Comment:
Why do we need the `tokio` feature? We aren't running a tokio runtime, and we would rather avoid code which assumes running in the tokio runtime context (it will panic if it tries to access stuff from tokio).

Reply:
The `tokio` feature was the only alternative to the `sync` feature, so it looked like using it was the only option for the async wasi, but now that I think about it, we might not need it here and just just the `sync` WASI - I'll test if the yielding works without it